src/databricks/labs/ucx/source_code/notebooks/cells.py

```diff
@@ -24,13 +24,20 @@
 
 class Cell(ABC):
 
-    def __init__(self, source: str, original_offset: int = 0):
+    def __init__(self, source: str, original_offset: int):
+        # The header line prevents a cell from having a zero offset.
+        if original_offset < 1:
+            raise ValueError("Cells must have a positive offset within the original file.")
         self._original_offset = original_offset
         self._original_code = source
         self._migrated_code = source
 
     @property
     def original_offset(self) -> int:
+        """Line offset of this cell within the original notebook file.
+
+        Example: if the line offset is 5, then line 10 within this cell corresponds to line 5 of the original file.
```

Comment:
Shouldn't this be the other way around? 

> then line 10 within this cell corresponds to line 5 of the original file.

should be

> then line 5 within this cell corresponds to line 10 of the original file.

Reply:
See: #1874